modules/configuring-default-route-timeouts-globally.adoc

```diff
@@ -12,15 +12,33 @@ You can configure the route timeouts globally by updating the `ROUTE_HAPROXY_TIM
 
 .Procedure
 
-. Set the value of `ROUTE_HAPROXY_TIMEOUT` in your subscription to your required timeout in seconds by running the following command:
+. Set the value of `ROUTE_HAPROXY_TIMEOUT` in your subscription to your required timeout in seconds by running the following command. Note that this causes pods to be redeployed in the `openshift-serverless` namespace.
 +
 .Setting the `ROUTE_HAPROXY_TIMEOUT` value to 900 seconds
 [source,terminal]
 ----
 $ oc patch subscription.operators.coreos.com serverless-operator -n openshift-serverless --type='merge' -p '{"spec": {"config": {"env": [{"name": "ROUTE_HAPROXY_TIMEOUT", "value": "900"}]}}}'
 ----
 +
-The `ROUTE_HAPROXY_TIMEOUT`  environment variable is managed by the Serverless Operator and by default is set to `600`. Set the value of `ROUTE_HAPROXY_TIMEOUT` in your subscription to your required timeout in seconds by running the following command. Note that this causes pods to be redeployed in the `openshift-serverless` namespace.
+Alternatively, you can set the value of `ROUTE_HAPROXY_TIMEOUT` in your subscription directly:
++
+.A subscription definition with `ROUTE_HAPROXY_TIMEOUT` set to 900 seconds
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+#...
+spec:
+  channel: stable
+  config:
+    env:
+      - name: ROUTE_HAPROXY_TIMEOUT
+        value: '900'
+#...
+----
++
+The `ROUTE_HAPROXY_TIMEOUT` environment variable is managed by the Serverless Operator and by default is set to `600`.
```

Comment:
I think "The `ROUTE_HAPROXY_TIMEOUT` environment variable is managed by the Serverless Operator and by default is set to `600`." should be placed in the introductory text before a customer hits the procedure steps.